src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -30,13 +39,17 @@
   </ItemGroup>
   <ItemGroup Condition="'$(IsPartialFacadeAssembly)' == 'true'">
     <ReferenceFromRuntime Include="System.Private.CoreLib" />
+    <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
```

Comment:
Why not use ReferenceFromRuntime?

Reply:
Do you mean something like this?
```C#
    <ReferenceFromRuntime Include="System.Runtime" />
    <!-- <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" /> -->
```

What would that do? I am already referencing System.Private.CoreLib.

Reply:
Never mind, if you are referencing implementation directly it has to be referenced to the Project directly and you've already referenced from runtime System.Private.CoreLib. That is correct, the only thing that is not correct is that all the ProjectReference that we add in a src project has to be to src projects not reference assembly.

So the reference below has to be a reference to the src project as well.

Reply:
right, got it. will fix that.